lapack.go

```diff
@@ -156,3 +156,15 @@ const (
 	AllEigVecMulQ  HowMany = 'B' // Compute all right and/or left eigenvectors multiplied by an input matrix.
 	SelectedEigVec HowMany = 'S' // Compute selected right and/or left eigenvectors.
 )
+
+// Job types for Dgeev.
+type (
+	JobLeftEV  byte
```

Comment:
Both jobs take the same values ('V' and 'N'), so if they are just the generic `lapack.Job`, I think it would be too easy for the caller to get them in the wrong order without any compiler error or runtime panic.